### PR TITLE
bump python version

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/thoth-application/issues/545#issuecomment-735908736

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ runtime_environments:
     operating_system: 
       name: rhel
```
Comment (bissenbay) [inline .thoth.yaml:8]: Should I change to `ubi`?

**Review by bissenbay — COMMENTED** [review]
@fridex I have made a change in `.thoth.yaml` and `Pipfile`, so this is exactly what I have been doing to others repos like changing `thoth.yaml`, `aicoe-ci.yaml` and `Pipfile`?

File: .thoth.yaml (hunk 1 above)
Comment (fridex) [reply .thoth.yaml:8]: No need to do that - rhel/ubi are basically synonyms here.

**Review by fridex — APPROVED** [review]
nice, thanks! :+1:

**Comment by bissenbay** [comment]
@harshad16 can you take a look, build is failing?

**Comment by harshad16** [comment]
@bissenbay , can you add the following file `.aicoe-ci.yaml`.
```
check:
  - thoth-pytest-py38
release:
  - upload-pypi-sesheta
```

More context: as storages is a module application we dont need to build it into a container image , we want instead the pipeline to release the module to PyPI on a tag release. 
Even though after adding the file the failed build-check won't disappear, anyhow it would not occur in other pull -request going forward.

**Comment by bissenbay** [comment]
> @bissenbay , can you add the following file `.aicoe-ci.yaml`.
> 
> ```
> check:
>   - thoth-pytest-py38
> release:
>   - upload-pypi-sesheta
> ```
> 
> More context: as storages is a module application we dont need to build it into a container image , we want instead the pipeline to release the module to PyPI on a tag release.
> Even though after adding the file the failed build-check won't disappear, anyhow it would not occur in other pull -request going forward.

thanks, https://github.com/thoth-station/storages/pull/2121/commits/db66d6a42127fa2a763ba6ad326bc577b4f26aec

**Comment by bissenbay** [comment]
> @bissenbay , can you add the following file `.aicoe-ci.yaml`.
> 
> ```
> check:
>   - thoth-pytest-py38
> release:
>   - upload-pypi-sesheta
> ```
> 
> More context: as storages is a module application we dont need to build it into a container image , we want instead the pipeline to release the module to PyPI on a tag release.
> Even though after adding the file the failed build-check won't disappear, anyhow it would not occur in other pull -request going forward.

@harshad16 thanks for the explanation in https://github.com/thoth-station/analyzer/pull/293/files, but I'm curious why we don't add in this repo `https://github.com/thoth-station/analyzer/pull/293/files` as we added it in https://github.com/thoth-station/analyzer/pull/293?

**Comment by harshad16** [comment]
@bissenbay sorry i didnt get your question? can you please re-phrase it for me

**Comment by harshad16** [comment]
merging this manually for now.

**Pull request merged**